omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java

```diff
@@ -552,4 +556,19 @@ public SimpleObject getAuditInfo(Person person) throws Exception {
 		ret.put("dateCreated", ConversionUtil.convertToRepresentation(person.getPersonDateCreated(), Representation.DEFAULT));
 		return ret;
 	}
+	
+	/**
+	 * Adds the link to patient resource in this person description if person is patient
+	 * 
+	 * @param description whose link set is to be modified
+	 * @return audit resource description
+	 */
+	private DelegatingResourceDescription ifPatientAddLink(DelegatingResourceDescription description) {
```

Comment:
Can we improve on the method name to reflect more meaning? eg. `addLinkIfPatient`, `addLinkToPatientIfRequired` etc. sounds better.